File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/7 - 9 and 10 March 2019/02_skeleton.py
minutes_control = int(input())
seconds_control = int(input())
length_slope = float(input())
seconds_for_100_meters = int(input())

total_control_in_seconds = minutes_control * 60 + seconds_control
time_slow = (length_slope / 120) * 2.5
marin_time = (length_slope / 100) * seconds_for_100_meters - time_slow

if marin_time <= total_control_in_seconds:
    print("Marin Bangiev won an Olympic quota!")
    print(f"His time is {marin_time:.3f}.")
else:
    print(f"No, Marin failed! He was {(marin_time - total_control_in_seconds):.3f} second slower.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/03. Online Exam - 9 and 10 March 2019/02. Skeleton.js
function skeleton(params) {
    let minutesRecord = Number(params.shift());
    let secondsRecord = Number(params.shift());
    let length = Number(params.shift());
    let secondsFor100meters = Number(params.shift());

    let record = (minutesRecord * 60) + secondsRecord;
    let timeCheat = (length / 120) * 2.5;
    let marinTime = (length / 100) * secondsFor100meters - timeCheat;

    if (marinTime <= record) {
        console.log(`Marin Bangiev won an Olympic quota!`);
        console.log(`His time is ${marinTime.toFixed(3)}.`);
    } else {
        console.log(`No, Marin failed! He was ${(marinTime - record).toFixed(3)} second slower.`);
    }
}


skeleton(['2', '12', '1200', '10']);  // Expected Output:
// Marin Bangiev won an Olympic quota!
// His time is 95.000.

skeleton(['1', '20', '1546', '12']);  // Expected Output:
// No, Marin failed! He was 73.312 second slower.
